ACTIONS.md

```diff
@@ -61,13 +61,13 @@ Text in **SUB bullets** is either for Resources Chapter (noted Chp 3) with addit
 *   [Recommend topics or speakers for Community Calls](#commcalls)
 *   [Help organize a Community Call](#commcalls)
 *   [Propose your idea for a Community Call summary post for our blog](#commcalls)
-    *   (Chp3 text) Take a look at examples of [community call posts](https://ropensci.org/tags/community-call/) in the [blog](#blog), consult the [blog guide](#blogguide) to see what’s involved then contact our Community Manager, Stefanie Butland, via [Slack](#channels-slack) or our [contact form](https://ropensci.org/contact/) to discuss your idea.
+    *   (Chp3 text) Take a look at examples of [community call posts](https://ropensci.org/tags/community-call/) in the [blog](#blog), consult the [blog guide](#blogguide) to see what’s involved then contact our Community Manager via [Slack](#channels-slack) or our [contact form](https://ropensci.org/contact/) to discuss your idea.
```

Comment:
should Community Manager link to ropensci.org/about#team? But now in the absence of community manager who would people contact?

Reply:
in the meantime actually we might want to completely remove the part about contacting the CM by slack as the next CM might choose a different workflow involving less slack DMs?

Reply:
Hmm, that's a good point, I'll remove it for now and can always add it back in later!